core/assets/bundles/bundle.properties

```diff
@@ -1115,6 +1115,7 @@ setting.bridgeopacity.name = Bridge Opacity
 setting.playerchat.name = Display Player Bubble Chat
 setting.showweather.name = Show Weather Graphics
 setting.hidedisplays.name = Hide Logic Displays
+setting.macnotch.name = Adapt interface to display notch
```

Comment:
If it's not a dynamic, add a tooltip `(provide a .tooltip entry)` that explains that it takes effect after restart.

Reply:
Thank you! Will do!

Reply:
Added `setting.macnotch.name.tooltip.tooltip` to all languages, but it's not fetched automatically by renderer. I don't think it's worth to adapt settings renderer for such a small tweak. Maybe I'll just do it like this?

```
setting.macnotch.name = Adapt interface to display notch (restart required)
```

Reply:
My bad, it's supposed to be `setting.macnotch.description`